docs/deployment/configuring_logging.md

```diff
@@ -0,0 +1,67 @@
+# Configuring the Logging stack via Gardenlet configurations
```

Comment:
Please add the document to the index (https://github.com/gardener/gardener/blob/master/docs/README.md#deployment)